covid_plot_vacs.py

```diff
@@ -287,6 +287,24 @@ def clean_vac_leg(label, first="1st Jab", second="2nd Jab"):
                         + 'PCR: Polymerase Chain Reaction',
               footnote_left=f'{source}Data Source: MOPH Covid-19 Dashboard,  CCSA Daily Briefing')
 
+    # Do a % of peak chart for cases vs. social distancingn (reduced mobility)
+    cols = ['Cases']
+    peaks = df[cols] / df[cols].rolling(7).mean().max(axis=0) * 100
+
+    mobility = import_csv("mobility", ['Date'])
+    
+    peaks = peaks.combine_first(mobility)
+    cols += ['Reduced Mobility Index (% of peak)']
+    legend = ["Confirmed Cases (% of peak)", "Reduced Mobility Index (% of peak)"]
```

Comment:
Maybe call it "Reduced Mobility Index - IHME (% of peak)"?

Reply:
Done